tool/tsh/app.go

```diff
@@ -266,6 +266,7 @@ func getRegisteredApp(cf *CLIConf, tc *client.TeleportClient) (app types.Applica
 	err = client.RetryWithRelogin(cf.Context, tc, func() error {
 		apps, err = tc.ListApps(cf.Context, &proto.ListResourcesRequest{
 			Namespace:           tc.Namespace,
+			ResourceType:        types.KindAppServer,
```

Comment:
Is there a reason why methods like `tc.ListApps` should no longer set `ResourceType` by themselves and instead expect the caller to set `ResourceType` on the request?